tests/test_index_order.py

```diff
@@ -32,7 +32,8 @@
     "dell_force10",
     "dell_powerconnect",
     "enterasys",
-    "extreme",
+    "extreme_slx",
+    "extreme_tierra",
```

Comment:
This will be a breaking change. Can we get some more details on this? One other thought that I do not believe I am in favor of, but want to have the conversation. Should these match the Netmiko driver names?

Reply:
From the contributing section of the readme.md for the project.

```
Naming

The template should be named using: {{ vendor_os }}_{{ command_with_underscores }}.textfsm

Ex: cisco_ios_show_cdp_neighbors.textfsm
Note: The vendor name must be valid from the [os_choices](https://github.com/networktocode/ntc-templates/blob/master/tests/test_index_order.py#L59) tests, which is primarily based on [Netmiko](https://github.com/ktbyers/netmiko/tree/master/netmiko) list of supported vendors. New vendors added should adhere to vendor_os.

Ex: vmware_nsx
```

Extreme has recently acquired (2018) the rights to multiple product lines (MLX, SLXOS, NOS (VDX platforms), VYOS) and introduced the a new OS as well (TierraOS).

As 'extreme'  does not follow the stated format, and there are no 'extreme' templates present, this change brings ntc-templates inline with netmiko w/ respect to Extreme's product lines. and solves an issue that the test_index_order.py has due to not matching the vendor_os correctly when ran. 

Reference: [https://github.com/ktbyers/netmiko/blob/develop/netmiko/ssh_dispatcher.py] lines 173 to 182. Arguably, the `extreme` can be kept for backwards compatibility, but it causes issues here when it attempts to check the order of templates due to not following the stated format for contributing .

```
    "extreme": ExtremeExosSSH,
    "extreme_ers": ExtremeErsSSH,
    "extreme_exos": ExtremeExosSSH,
    "extreme_netiron": ExtremeNetironSSH,
    "extreme_nos": ExtremeNosSSH,
    "extreme_slx": ExtremeSlxSSH,
    "extreme_tierra": ExtremeTierraSSH,
    "extreme_vdx": ExtremeNosSSH,
    "extreme_vsp": ExtremeVspSSH,
    "extreme_wing": ExtremeWingSSH,
```

Reply:
Ok, found a working solution, by putting the correctly formated vendor_os pairs ahead of the 'extreme' entry, the ordering test works correctly.  The other changes should be done as well.